src/components/GridItems/GridItems.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import CoffeeItem from "../CoffeeItem/CoffeeItem";
+import gridItems from "../../data/gridItems.js";
+import "./GridItems.css";
+
+class GridItems extends Component {
+  state = {
+    items: Array(8).fill(0),
```

Comment:
Add JSON file for the grid items 
```suggestion
    items: coffeeItems,
```